src/app/dim-ui/usePopper.ts

```diff
@@ -125,6 +126,13 @@ export function usePopper({
   /** Is this placed on a fixed item? Workaround for https://github.com/popperjs/popper-core/issues/1156. TODO: make a "positioning context" context value for this */
   fixed?: boolean;
   padding?: Padding;
+
+  /**
+   * An optional state variable. Use this to have popper recalculate the position of
+   * a popover when conditionally rendering the contents of the popover (thus
+   * causing the dimensions to potentially change).
+   */
+  state?: boolean;
```

Comment:
Open to other names here. Wasn't sure what to go with, so just used "state" for now.